test_scripts/API/VehicleData/OnVehicleData/008_OnVD_enum_and_bool_values.lua

```diff
@@ -0,0 +1,38 @@
+----------------------------------------------------------------------------------------------------
+-- Description: Check that SDL processes OnVehicleData notification with <vd_param> parameter
+-- Positive cases for all possible values for Enum and Boolean VD parameters and sub-parameters
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) SubscribeVehicleData, OnVehicleData RPCs and <vd_param> parameter are allowed by policies
+-- 3) App is registered
+-- 4) App is subscribed to <vd_param> parameter data
+--
+-- In case:
+-- 1) HMI sends valid OnVehicleData notification with <vd_param> parameter data to SDL
+-- (iterate trough all possible enum and boolean values of applicable parameters)
+-- SDL does:
+-- - a) transfer this notification to App
+-- Exception: Notification for unsubscribable VD parameter is not transfered
+----------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/common')
+
+--[[ Local Constants ]]
+local testTypes = {
+  common.testType.ENUM_ITEMS,
+  common.testType.BOOL_ITEMS
+}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment and update preloaded_pt file", common.preconditions)
+common.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("Activate App", common.activateApp)
+
+common.Title("Test")
+common.getTestsForOnVD(testTypes)
+
+common.Title("Postconditions")
+common.Step("Stop SDL", common.postconditions)
```

Comment:
```suggestion
common.runner.Step("Stop SDL", common.postconditions)
```
I think it would be better to use the runner as a component of the common module, rather than just taking functions directly from the runner.
Calling script runner functions via the common module is confusing

Reply:
Done in 53d0806